seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -0,0 +1,135 @@
+import json
+import mock
+
+from django.urls.base import reverse
+
+from seqr.models import Project
+from seqr.views.apis.anvil_workspace_api import anvil_workspace_page, create_project_from_workspace
+from seqr.views.utils.test_utils import AnvilAuthenticationTestCase
+
+WORKSPACE_NAMESPACE = 'my-seqr-billing'
+EXIST_WORKSPACE_NAME = 'anvil-project 1000 Genomes Demo'
+WORKSPACE_NAME = 'anvil project name'
+NEW_WORKSPACE_NAME = 'anvil new project'
+
+LOAD_SAMPLE_DATA = [
+    ["Family ID", "Individual ID", "Previous Individual ID", "Paternal ID", "Maternal ID", "Sex", "Affected Status",
+     "Notes", "familyNotes"],
+    ["1", "NA19675", "NA19675_1", "NA19678", "", "Female", "Affected", "A affected individual, test1-zsf", ""],
+    ["1", "NA19678", "", "", "", "Male", "Unaffected", "a individual note", ""],
+    ["21", "HG00735", "", "", "", "Female", "Unaffected", "", "a new family"]]
+
+BAD_SAMPLE_DATA = [["1", "NA19674", "NA19674_1", "NA19678", "NA19679", "Female", "Affected", "A affected individual, test1-zsf", ""]]
+
+REQUEST_BODY = {
+            'genomeVersion': '38',
+            'uploadedFileId': 'test_temp_file_id',
+            'description': 'A test project',
+            'agreeSeqrAccess': True,
+        }
+
+
+@mock.patch('seqr.views.utils.permissions_utils.user_get_workspace_access_level')
+class AnvilWorkspaceAPITest(AnvilAuthenticationTestCase):
```

Comment:
you need to also add another set of test cases that check that everything fails in local-only mode. I wouldn't do it the way we test in other files by making a mixin that you use for the different modes, I would just make a separate one that has a single function to test that all the urls error out
```
class AnvilWorkspaceAPILocalTest(AuthenticationTestCase):
  def test_no_local_access(self):
    ...
```

Reply:
Added a new test class `NoGoogleAnvilWorkspaceAPITest`.

Reply:
Should we remove these URLs completely in the local-only mode? Because it will always redirect to the login page. But it should remind the user that the URLs are not working.

Reply:
No, we should include them and just always have them redirect.Keeping our routing simpler is better and theres no harm